### ISSUE TITLE
Contact document owners Phase 2 - 508 Compliance

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#958]

**Comment by cthomasONRR** [comment]
@Maroyafaied Based on the accessibility tracking spreadsheet, here are the 9 files still left in Phase 2:

![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/5d8be918-59ae-4d7d-9f3c-3d9f7c90c4c1)

**Comment by Maroyafaied** [comment]
@cthomasONRR Awesome, thank you!

**Comment by Maroyafaied** [comment]
@cthomasONRR @mcharg I'm looking on what's left in phase 2 and it looks like some of the titles are not correct anymore. I can fix it manually but wondering if it's something with the formatting? ![image.png](https://images.zenhubusercontent.com/5c58acad07cc4d6f39e54f67/a49b74a2-fc62-4fd2-ab70-2306492bada5)

**Comment by mcharg** [comment]
@Maroyafaied It should be pulling the title from the content audit sheet.  I'll take a look.

**Comment by cthomasONRR** [comment]
Oh weird... It looks like when I sorted by Program Manager, than the entire table didn't conform to the sort.

**Comment by mcharg** [comment]
Fixed the titles

**Comment by cthomasONRR** [comment]
@mcharg Thanks!!!
